meta-lmp-bsp/recipes-bsp/alsa/alsa-state-stm32mp1/system-generator-alsa-states

```diff
@@ -0,0 +1,66 @@
+#!/bin/sh
+
+# Script which detect automaticaly the alsa-states file to put
+# in place in fuonction of board on which the script are executed.
```

Comment:
```suggestion
# in place in function of board on which the script are executed.
```

Reply:
Since this is a copy of the original script available in the meta-stm repo, I think it is ok to keep the typo here to reduce the diff.